### PR TITLE
[C-2705] Port upload files page to new upload form

### PR DESCRIPTION
**Pull request by amendelsohn**
### Description

Screenshot 2023-06-01 at 5 17 01 PM

### Feature Flags ###

`UPLOAD_REDESIGN_ENABLED`

### CODE REVIEW

File: packages/web/src/pages/upload-page/UploadPageNew.tsx [hunk 1]
```diff
@@ -1,23 +1,159 @@
...
+
   return (
     <Page
       title='Upload'
       description='Upload and publish audio content to the Audius platform'
       contentClassName={styles.upload}
       header={<Header primary={'Upload'} />}
     >
-      This is the new upload page
+      <SelectPage
```
Comment (dylanjeffers) [inline packages/web/src/pages/upload-page/UploadPageNew.tsx:141]: sweet, up to you but i would to move all these props down to the leaf compnonets eventually. what do you think?

**Review by dylanjeffers — APPROVED** [review]
good looks, what do you htink about last comment?

File: packages/web/src/pages/upload-page/UploadPageNew.tsx (hunk 1 above)
Comment (amendelsohn) [reply packages/web/src/pages/upload-page/UploadPageNew.tsx:141]: Yes, I plan to. Just wanted to get a piece working for now 😄

Comment (amendelsohn) [reply packages/web/src/pages/upload-page/UploadPageNew.tsx:141]: It wasn't a class component yet so I'll port the SelectPage and push the hooks, state, and callbacks down. I think in the end we'll want the shared page, form, and track state up here and the rest a level down

Comment (dylanjeffers) [reply packages/web/src/pages/upload-page/UploadPageNew.tsx:141]: dope thanks for context!

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/amendel-upload-files-page

**Pull request merged**